src/OpenLoco/src/Objects/DockObject.cpp

```diff
@@ -99,4 +99,10 @@ namespace OpenLoco
 
         return std::span<const std::uint8_t>(partsPointer, end);
     }
+
+    std::span<const BuildingPartAnimation> DockObject::getBuildingPartAnimations() const
+    {
+        return std::span<const BuildingPartAnimation>(buildingPartAnimations, numBuildingParts);
+    }
```

Comment:
Not used yet, right? Perhaps intended for another PR?

Reply:
Its used in a templated function so its required to compile